UI/backend/app.py

```diff
@@ -1,7 +1,11 @@
 from flask import Flask, request, jsonify
 from flask_cors import CORS
-import subprocess
-import os
+from constants import UniJsonKeys
+from db_utils import connect_to_db, insert_time_slot, insert_course, insert_instructor
+import requests
```

Comment:
change import order.
First should be imports of the following form:

import request

after them should be imports of the following form:

from constants import UniJsonKeys

Reply:
done